project_model_to_task/models/project.py

```diff
@@ -10,14 +10,14 @@
 class ProjectTask(models.Model):
     _inherit = 'project.task'
 
+    @api.multi
```

Comment:
multi is the default mode in v10 (dixit A Lesuisse). Easier to read with few decorator